Libplanet/Store/IStore.cs

```diff
@@ -217,42 +216,53 @@ void PutBlock<T>(Block<T> block)
         /// <c>a = 1; b = 2; c = 1</c> (all states) nor
         /// <c>b = 1; c = 1</c> (delta).</para>
         /// </remarks>
-        IImmutableDictionary<Address, IValue> GetBlockStates(HashDigest<SHA256> blockHash);
+        /// <seealso cref="SetBlockStates"/>
+        IImmutableDictionary<string, IValue> GetBlockStates(HashDigest<SHA256> blockHash);
 
+        /// <summary>
+        /// Sets the states updated by actions in the specified block
+        /// (i.e., <paramref name="blockHash"/>).
+        /// </summary>
+        /// <param name="blockHash"><see cref="Block{T}.Hash"/> to update states.
+        /// </param>
+        /// <param name="states">The states updated by actions in the specified block
+        /// (i.e., <paramref name="blockHash"/>).
+        /// </param>
+        /// <seealso cref="GetBlockStates"/>
         void SetBlockStates(
             HashDigest<SHA256> blockHash,
-            IImmutableDictionary<Address, IValue> states
+            IImmutableDictionary<string, IValue> states
         );
 
-#pragma warning disable MEN002
         /// <summary>
         /// Looks up a state reference, which is a block's <see cref="Block{T}.Hash"/> that contains
-        /// an action mutating the <paramref name="address"/>' state.
+        /// an action mutating the <paramref name="key"/>' state.
```

Comment:
I'm not sure, but in my thought, it seems gone the reason to omit *s*.

```suggestion
        /// an action mutating the <paramref name="key"/>'s state.
```

Reply:
Fixed and amended the commit: 7e100f863e404cda983c05f0baa90905d2a8ecdd.